src/js/backup_restore.js

```diff
@@ -144,8 +144,13 @@ function configuration_backup(callback) {
                 configuration.LED_STRIP = jQuery.extend(true, [], FC.LED_STRIP);
                 configuration.LED_COLORS = jQuery.extend(true, [], FC.LED_COLORS);
                 configuration.BOARD_ALIGNMENT_CONFIG = jQuery.extend(true, {}, FC.BOARD_ALIGNMENT_CONFIG);
-                configuration.CRAFT_NAME = FC.CONFIG.name;
-                configuration.DISPLAY_NAME = FC.CONFIG.displayName;
+                if (semver.gte(FC.CONFIG.apiVersion, API_VERSION_1_45)) {
+                    configuration.CRAFT_NAME = FC.CONFIG.craftName;
+                    configuration.PILOT_NAME = FC.CONFIG.pilotName;
+                } else {
+                    configuration.CRAFT_NAME = FC.CONFIG.name;
+                    configuration.DISPLAY_NAME = FC.CONFIG.displayName;
+                }
```

Comment:
Dunno, might be a stupid suggestion. But given that we have different names for fields, we could avoid version check by doing something like: 
```suggestion  
                configuration.CRAFT_NAME = FC.CONFIG.craftName ?? FC.CONFIG.name;
                configuration.PILOT_NAME = FC.CONFIG.pilotName ?? '';
                configuration.DISPLAY_NAME = FC.CONFIG.displayName ?? '';
```

But this is up for discussion though.